client/src/App.js

```diff
@@ -1,6 +1,7 @@
 import "./App.css";
 import BottomNav from "./components/BottomNav";
 import Header from "./components/Header";
+import Home from "./pages/Home";
 import { Switch, Route, Redirect } from "react-router";
 
 export default function App() {

```

Comment:
I can not comment somehow in the Switch section. You used the exact path in nearly every Route but it should also work without exact, right? I think you will move the main element inside the component like you've done in your Home component, right?